src/cloud-functions.ts

```diff
@@ -474,6 +474,19 @@ export function makeCloudFunction<EventData>({
     },
   });
 
+  if (options.schedule) {
+    cloudFunction.__requiredAPIs = [
+      {
+        api: 'pubsub.googleapis.com',
```

Comment:
I wonder if we need to declare pubsub here. It's a core API that is always enabled with GCF and would presumedly be enabled in any future services.